docs/environments/icp.md

```diff
@@ -0,0 +1,49 @@
+# IBM Cloud Private Deployment Guide
+
+Federation v2 can be deployed to and manage [IBM Cloud Private](https://www.ibm.com/cloud/private) clusters.
+As Federation v2 requires Kubernetes v1.11 or greater, please make sure to deploy IBM Cloud Private 3.1.1
+or higher.
+
+The following example deploys two IBM Cloud Private 3.1.1 clusters named `cluster1` and `cluster2`.
+
+## Install IBM Cloud Private
+
+Please follow the [guide in IBM Cloud Private 3.1.1 Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install.html)
+to install.
+
+**NOTE:** As we need to install two clusters named `cluster1` and `cluster2`, so after `cluster/config.yaml`
+is generated, update the names of the 2 clusters to 'cluster1' and 'cluster2' before installing federation.
```

Comment:
(No action required) Should this be \`cluster1\` and \`cluster2\` for consistency?